public/css/primitives.css

```diff
@@ -1,89 +1,110 @@
 :root {
-	--complete-color: #36B37E; /* G300 - Fine pine */ 
-	--ontrack-color: #4C9AFF; /* B100 - Arvo breeze */
-	--behind-color: #FFC400; /* Y200 - Pub mix */
-	--last-period-behind-color: #FFE380; /*Y100 - Cowbell*/
-	--warning-color: #ff9d00; /* CUSTOM, was Y300 - Golden state */
-	--last-period-warning-color: #c78a27;
-	--unknown-color: #dab092;
-	--notstarted-color: #97A0AF; /* N80 - Spooky ghost */
-	--blocked-color: #f2000b;
-	--last-period-blocked-color: #b0282f;
-	--ahead-color: #79E2F2; /* T100 - Hairy fairy */
-	--ahead-dark-color: #00A3BF;
-	--new-color: #6554C0;
-	--white-color: rgb(255 255 255);
-	--black-color: rgb(0 0 0);
-	--slate-900-color: rgb(17 24 39);
-	--slate-400-color: #505F79;
-	--slate-300-color: #E7E7E7;
-	--gray-200-color: #E5E7EB;
-	--gray-100-color: #ebecf0; /* N30 - Northeast snow */
-	--gray-20-color: #F4F5F7; /* N20 - Gram's hair */
-	--gray-10-color: #F4F5F7; /* N20 - Gram's hair */
-	--sky-100-color: rgb(224 242 254);
-
-	--bitovi-red-orange: #F5532D;
-
-
-	/* design system, for Atlassian */
-	--ds-icon-success: var(--complete-color);
-	--ds-icon-warning: var(--warning-color);
-	--ds-icon-danger: var(--blocked-color);
-	--ds-text-success: #36885e;
-	--ds-text-warning: var(--last-period-warning-color);
-	--ds-text-danger: var(--last-period-blocked-color);
+  --complete-color: #22a06b;
```

Comment:
Could you add a comment for which color this is from the design system?

Reply:
yeah

Reply:
Ah the new colors don't have names